pkg/arvo/app/permission-group-hook.hoon

```diff
@@ -67,22 +65,19 @@
   =/  permissions
     %-  silt
     %+  turn  ~(tap in permission-paths)
-    |=  [=path =kind]
-    path
+    |=([=path =kind] path)
```

Comment:
Rather than turn set into a list and then back into a set you can use run:in

Reply:
Thanks! Great idea. Will do.